tests/sample_data/hiero_xml_export.xml

```diff
@@ -0,0 +1,132 @@
+            <in>0.0</in>
+            <out>1055.0</out>
+            <start>14</start>
+            <end>1069</end>
+            <file id="A160C005_171213_R0MN">
+              <name>A160C005_171213_R0MN</name>
+              <pathurl>file://localhost/A160R0MN/A160R0MN/A160C005_171213_R0MN.mxf</pathurl>
+              <duration>1055</duration>
+              <rate>
+                <ntsc>FALSE</ntsc>
+                <timebase>24</timebase>
+              </rate>
+              <width>3424</width>
+              <height>2202</height>
+              <media>
+                <video>
+                  <duration>1055</duration>
+                  <samplecharacteristics>
+                    <width>3424</width>
+                    <height>2202</height>
+                  </samplecharacteristics>
+                </video>
+              </media>
+              <timecode>
+                <rate>
+                  <ntsc>FALSE</ntsc>
+                  <timebase>24</timebase>
+                </rate>
+                <frame>1101071</frame>
+                <displayformat>NDF</displayformat>
+                <string>12:44:37:23</string>
+              </timecode>
+            </file>
+          </clipitem>
+          <clipitem>
+            <name>/</name>
+            <duration>400.0</duration>
+            <rate>
+              <ntsc>FALSE</ntsc>
+              <timebase>24</timebase>
+            </rate>
+            <enabled>TRUE</enabled>
+            <in>326400.0</in>
+            <out>326639.0</out>
+            <start>1085</start>
+            <end>1324</end>
+            <file id="/">
+              <name>/</name>
+              <duration>1</duration>
+              <rate>
+                <ntsc>FALSE</ntsc>
+                <timebase>24</timebase>
+              </rate>
+              <timecode>
+                <rate>
+                  <ntsc>FALSE</ntsc>
+                  <timebase>24</timebase>
+                </rate>
+                <frame>0</frame>
+                <displayformat>NDF</displayformat>
+                <string>00:00:00:00</string>
+              </timecode>
+            </file>
+            <filter>
+              <effect>
+                <name>Time Remap</name>
+                <effectid>timeremap</effectid>
+                <effectcategory>motion</effectcategory>
+                <effecttype>motion</effecttype>
+                <mediatype>video</mediatype>
+                <parameter>
+                  <name>speed</name>
+                  <parameterid>speed</parameterid>
+                  <value>0.25</value>
+                </parameter>
+              </effect>
+            </filter>
+          </clipitem>
+        </track>
+      </video>
+      <audio>
+        <format>
+          <samplecharacteristics>
+            <depth>16</depth>
+            <samplerate>48000</samplerate>
+          </samplecharacteristics>
+        </format>
+      </audio>
+    </media>
+  </sequence>
+</xmeml>
```

Comment:
Would you mind adding a unit test that verifies that this file reads correctly? It just needs to read the file in and check for a few properties.

thanks!

Reply:
Sure!
I'll look at similar tests and create one for the Hiero XML.